Language/Functions/Bits and Bytes/bitRead.adoc

```diff
@@ -38,7 +38,31 @@ The value of the bit (0 or 1).
 
 --
 // OVERVIEW SECTION ENDS
+=== Example Code
+// Describe what the example code is all about and add relevant code
+Reads the bit at the specified position and returns the value. In this example the binary representation of 6 is 0110, since `n=1` the value of the second bit from the right (which is 1 in this case) will be read and printed. 
+[source,arduino]
+----
+void setup() 
+{
+  Serial.begin(9600); 
```

Comment:
```suggestion
  Serial.begin(9600);
  while (!Serial) {
    ; // wait for serial port to connect. Needed for native USB port only
  }
```
This will make the example more friendly to users of the native USB boards (e.g., Leonardo).